pyanaconda/ui/gui/spokes/welcome.py

```diff
@@ -62,7 +62,6 @@ class WelcomeLanguageSpoke(LangLocaleHandler, StandaloneSpoke):
     def __init__(self, *args, **kwargs):
         StandaloneSpoke.__init__(self, *args, **kwargs)
         LangLocaleHandler.__init__(self)
-        self._xklwrapper = XklWrapper.get_instance()
```

Comment:
Given this is a singleton I wonder if we don't instantiate it here to do it early enough (even though it will be used only from keyboard spoke). The XklWrapper class doc says that initialization takes quite a lot of time.
But even if it is the case we could probably initialize it more explicitily, not just as a side effect of initializinng welcome spoke.

Reply:
Good point. However, the instance was used before this commit https://github.com/rhinstaller/anaconda/commit/4e73305378a39e6b0db677ed48cb35ed8709b2e4. So it really looks like a leftover.